doc/source/whatsnew/v0.20.0.txt

```diff
@@ -792,6 +792,7 @@ Performance Improvements
 Bug Fixes
 ~~~~~~~~~
 
+- Bug in ``DataFrame.values`` now returns an ``numpy.ndarray`` of ``pandas.Timestamp`` for tz-aware columns; previously this returned ``pandas.DateTimeIndex`` (:issue:`14052`)
```

Comment:
say ``object`` dtyped numpy array of ``Timestamp`` you don't need to say ``pandas.*`` that's implied.